src/audio/module_adapter/module_adapter.c

```diff
@@ -179,7 +193,14 @@ int module_adapter_prepare(struct comp_dev *dev)
 	int i = 0;
 
 	comp_dbg(dev, "module_adapter_prepare() start");
+#if CONFIG_IPC_MAJOR_4
+	/* verify params */
```

Comment:
I think comments should add information, not obvious from the code, and a comment like this above a call to `comp_verify_params()` doesn't seem to add a lot of information. Instead maybe clarify that that function not only verifies parameters, but does more as discussed above